packages/ui/src/components/va-input/components/VaInputWrapper/_variables.scss

```diff
@@ -3,7 +3,7 @@
   --va-input-wrapper-background: var(--va-background, #f5f9fb);
   --va-input-wrapper-background-opacity: 1;
   --va-input-wrapper-border-color: var(--va-input-bordered-color);
-  --va-input-wrapper-min-width: 280px;
+  --va-input-wrapper-min-width: 240px;
```

Comment:
@rustem-nasyrov . I changed min width to 240px so it perfectly fit DatePicker in DateInput.

Do you see anything wrong here?

Reply:
No, I don't.